src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -416,3 +418,25 @@ export function getTileName(x: number, y: number, gridSize: GridSize): string {
   const tileId = `${`${tileY}`.padStart(nbDigits, '0')}${`${tileX}`.padStart(nbDigits, '0')}`;
   return `${sheetCode}_${gridSize}_${tileId}`;
 }
+
+/**
+ * Verify if a TIFF contains only 8 bits bands.
+ *
+ * @param tiff
+ * @returns true if 8 bits TIFF
+ */
+export async function is8BitsTiff(tiff: Tiff): Promise<boolean> {
+  const baseImage = tiff.images[0];
+  if (baseImage === undefined) throw new Error("Can't get base image");
```

Comment:
Do we need more info to debug this? For example, when this is triggered, would the user trivially understand which TIFF this applied to?